classes/class-wc-connect-api-client.php

```diff
@@ -291,13 +291,15 @@ protected function request( $method, $path, $body = array() ) {
 				return $headers;
 			}
 
+			$http_timeout = 60; // 1 minute
+			wc_set_time_limit( $http_timeout + 10 );
```

Comment:
Hummm... you're right. I've been assuming everywhere that WooCommerce is available. I wonder in how many ways our plugin 💥 when you deactivate WooCommerce.